app/utils/nameHelpers.js

```diff
@@ -74,3 +76,17 @@ function checkState(name, expectedState) {
 
   return info.state === expectedState;
 }
+
+export const formatName = name => {
+  if (!name)
+    return name;
+
+  try {
+    const unicode = punycode.toUnicode(name);
+    if (unicode !== name) {
+      return `${name}/ (${unicode})`;
+    }
+  } catch(e) {}
```

Comment:
I think the catch should probably write exceptions to the debug log, and maybe add something to the telemetry feedback to kyokan.

Reply:
Some names are not punycode and they can break the `.toUnicode` method. No big deal, they are not really punycode. This is why there's no track of this error.

Reply:
i thought it might be useful for identifying any edge cases where the library doesn't recognize newer unicode, or things like that.